src/action-controller/session.cr

```diff
@@ -22,6 +22,9 @@ class ActionController::Session < Hash(String, String | Int64 | Float64 | Bool)
   getter modified : Bool
   property domain : String?
   @encoder : MessageEncryptor | MessageVerifier
+  @store : Hash(String, String | Int64 | Float64 | Bool)
+
+  forward_missing_to @store
 
   def initialize
     super
```

Comment:
I think we can remove this call to `super`

Reply:
Good catch!